### PR TITLE
Functional tests - Refacto 'FO > User account > Order history > Consult order list' test

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto 'FO > User account > Order history > Consult order list' test
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts [hunk 1]
```diff
@@ -0,0 +1,190 @@
+import {expect} from 'chai';
+import type {BrowserContext, Page} from 'playwright';
+
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts]:
```suggestion
```

File: tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+import foMyAccountPage from '@pages/FO/myAccount';
+
+// Import data
+import {Statuses} from '@data/demo/orderStatuses';
+import {PaymentMethods} from '@data/demo/paymentMethods';
+import AddressFaker from '@data/faker/address';
+import CustomerFaker from '@data/faker/customer';
+import {Order, OrderHistory} from '@data/types/order';
+import {Products} from '@data/demo/products';
+
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts:25]:
```suggestion

import {expect} from 'chai';
import type {BrowserContext, Page} from 'playwright';

```

File: tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    email: customerData.email,
+    country: 'France',
+  });
+  const orderData: Order = {
+    customer: customerData,
+    productId: 1,
+    productQuantity: 1,
+    paymentMethod: PaymentMethods.wirePayment.moduleName,
+  };
+
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts]:
```suggestion
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/03_userAccount/02_orderHistory/01_consultOrderList.ts]: done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**